Cargo.toml

```diff
@@ -27,10 +27,16 @@ defmt = ["dep:defmt"]
 # Supports serde
 serde = ["dep:serde"]
 
+borsh = ["dep:borsh"]
+
+schema = ["dep:schemars", "std"]
```

Comment:
Any reason not to call this schemars instead?

Reply:
will name schemars.